concent_api/core/payments/backends/mock.py

```diff
@@ -52,7 +52,15 @@ def cover_additional_verification_cost(
 
 
 def call_on_confirmed_transaction(
-    _tx_hash: str,
-    _callback: Callable,
+    tx_hash: str,  # pylint: disable=unused-argument
+    callback: Callable,  # pylint: disable=unused-argument
+) -> None:
+    pass
+
+
+def validate_that_last_closure_time_is_older_than_oldest_payment(
```

Comment:
It shouldn't be here anymore.

Reply:
I think it should stay. This function calls `get_list_of_payments` and in unit tests we don't want to call `get_list_of_payments`, just mock it and pass validation

Reply:
But the purpose of this backend is different. This is mocked backend, it must be consistent with sci backend. If you want to create a reusable mock place it in `tests/utils.py`.

Reply:
removed